jetstream/kv.go

```diff
@@ -67,7 +67,10 @@ type (
 		// WatchAll will invoke the callback for all updates.
 		WatchAll(ctx context.Context, opts ...WatchOpt) (KeyWatcher, error)
 		// Keys will return all keys.
+		// DEPRECATED: Use ListKeys instead to avoid memory issues.
```

Comment:
which was the memory issue?

Reply:
Have a use case now with > 600k keys.

Reply:
ah I see, makes sense to switch to use channels instead